dataprofiler/profilers/data_labeler_column_profile.py

```diff
@@ -27,15 +27,21 @@ def __init__(self, name, data_labeler_dirpath=None, options=None):
             if not isinstance(options, DataLabelerOptions):
                 raise ValueError("DataLabelerColumn parameter 'options' must be"
                                  " of type DataLabelerOptions.")
-            if options.data_labeler_dirpath:
-                data_labeler_dirpath = options.data_labeler_dirpath
             if options.max_sample_size:
                 self._max_sample_size = options.max_sample_size
 
-        self.data_labeler = DataLabeler(
-            labeler_type='structured',
-            dirpath=data_labeler_dirpath,
-            load_options=None)
+        self.data_labeler = None
+        if options and options.data_labeler_object:
+            self.data_labeler = options.data_labeler_object
+        if self.data_labeler is None:
+            data_labeler_dirpath = None
+            if options and options.data_labeler_dirpath:
```

Comment:
I don't believe the`options.data_labeler_dirpath` in the if statement is necessary as at worst case it sets it to `None` which it already would be. so maybe just `if options:`

Reply:
fixed